src/FlightDisplay/FlyViewWidgetLayer.qml

```diff
@@ -138,9 +138,21 @@ Item {
         // Width is difficult to access directly hence this hack which may not work in all circumstances
         property real leftEdgeBottomInset:  visible ? bottomEdgeLeftInset + width/18 - ScreenTools.defaultFontPixelHeight*2 : 0
         property real rightEdgeBottomInset: visible ? bottomEdgeRightInset + width/18 - ScreenTools.defaultFontPixelHeight*2 : 0
+        property real rootWidth:            _root.width
+        property var  itemX:                virtualJoystickMultiTouch.x   // real X on screen
+
+        onRootWidthChanged: virtualJoystickMultiTouch.status == Loader.Ready && visible ? virtualJoystickMultiTouch.item.uiTotalWidth = rootWidth : undefined
+        onItemXChanged:     virtualJoystickMultiTouch.status == Loader.Ready && visible ? virtualJoystickMultiTouch.item.uiRealX = itemX : undefined
 
         //Loader status logic
-        onLoaded:           virtualJoystickMultiTouch.visible ?  virtualJoystickMultiTouch.item.calibration = true : virtualJoystickMultiTouch.item.calibration = false
+        onLoaded: {
+            if(virtualJoystickMultiTouch.visible) {
+                virtualJoystickMultiTouch.item.calibration = true 
+                virtualJoystickMultiTouch.item.uiTotalWidth = rootWidth
+                virtualJoystickMultiTouch.item.uiRealX = itemX
+            }
+            else virtualJoystickMultiTouch.item.calibration = false
```

Comment:
This all looks good, thanks so much. Could you update to match the coding style though:
```
if () {
   ... // Braces always used even if single line of code to prevent problems further down the road when a new line is added
} else { // else doesn't take a new line and follows brace
   ... // Braces always used even if single line of code to prevent problems further down the road when a new line is added
}

Reply:
Hi, thank you so much for your revision.
You are completely right, I did not match the coding style, I will solve this as soon as I can. Probably next Sunday :) 

Thank you for your time.

Reply:
Hi @HTRamsey  I'm really sorry for not completely solve that coding style problem but my PC literally blew up the other day, as I'm a student I didn't have other way to code.
Right now I have check your commits, everything okay, hope that my setback haven't taken much of your time to solve. 

PD: Looking forward for helping in other stuff that you request, this time with a brand new PC haha.

Reply:
Oof sorry about the PC! And no worries I 100% should have asked first but it was something small and people get busy. Thanks for doing this!